libs/blocks/quiz/quiz.js

```diff
@@ -258,6 +258,10 @@ const App = ({
     if (fragmentURL) {
       loadFragments(fragmentURL);
     }
+    const iconBg = getStringValue('icon-background-color');
+    if (iconBg) {
+      document.documentElement.style.setProperty('--quiz-icon-bg', iconBg);
```

Comment:
This could be causing re-rendering in the browser, would it make sense to scope it lower to the component instead of as an inline style on the HTML el?

Reply:
The srcset idea does seem like a good one. As to the backgrounds, each button can be identified by its data-option-name attribute, so perhaps logic similar to what would write the picture srcset could also write a style block for the backgrounds.

Reply:
Changed scope of background and added srcset